includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -641,6 +662,32 @@ protected static function get_twentyseventeen_navigation_outer_height() {
 		return 72;
 	}
 
+	/**
+	 * Add required styles for featured image header and image blocks in Twenty Twenty.
+	 */
+	public static function add_twentytwenty_masthead_styles() {
+		add_action(
+			'wp_enqueue_scripts',
+			static function() {
+				ob_start();
+				?>
+				<style>
+				.featured-media amp-img {
+					position: static;
+				}
+
+				.wp-block-image img {
+						display: block;
```

Comment:
Extra indentation here.
```suggestion
					display: block;
```